unlock_cond_expiration.go

```diff
@@ -28,6 +28,11 @@ func (s *ExpirationUnlockCondition) VBytes(rentStruct *RentStructure, _ VBytesFu
 		s.ReturnAddress.VBytes(rentStruct, nil)
 }
 
+func (s *ExpirationUnlockCondition) WorkScore(workScoreStructure *WorkScoreStructure) WorkScore {
+	// Expiration condition requires Commitment inputs to be loaded and signature checks, but this is taken care of when the output is consumes, not created.
+	return workScoreStructure.FactorData.Multiply(s.Size())
```

Comment:
But when do we charge for that?

Reply:
This is taken into account by the cost of the unlocks and the cost of the ContextInputs provided to unlock this output.